hack/lib/common-var-init.sh

```diff
@@ -178,7 +178,25 @@ API_SECURE_PORT=${API_SECURE_PORT:-6443}
 # WARNING: For DNS to work on most setups you should export API_HOST as the docker0 ip address,
 API_HOST=${API_HOST:-"${lohostname}"}
 API_HOST_IP=${API_HOST_IP:-"0.0.0.0"}
-API_HOST_IP_EXTERNAL=${API_HOST_IP_EXTERNAL:-"${hostip}"}
+
+# Fix the issue 1211 at https://github.com/CentaurusInfra/arktos/issues/1211
+API_HOST_IP_EXTERNAL=${API_HOST_IP_EXTERNAL:-}
+if [ -z ${API_HOST_IP_EXTERNAL} ]
+then
+  if [ -f "/etc/netplan/50-cloud-init.yaml" ]; then
```

Comment:
this is not a generic approach and it will fail on some machines. e.g. on my GCE VM, this file does not exist.

Reply:
Currently the bug is found in AWS Ubuntu 18.05 for Mizar. so we can fix the issue for this case on AWS Ubuntu 18.05. If your machine is GCE VM and the file does not exist, the script gives warning and exit. You have to manually set the parameter API_HOST_IP_EXTERNAL.

Reply:
we cannot fix one env but break another one that worked before.

Reply:
I make the more generic changes based on your suggestion to support Ubuntu 16.04 and GCE VM.